commands/help.py

```diff
@@ -25,12 +25,14 @@ async def help_command(ctx, client):
     _embedMessage.add_field(helpMessage, commandPrefix2 + "adminverify @Mention",
                             "Manually verifies a user. Note that this does NOT add in a role and simply adds them to the database\nEx: " + commandPrefix2 + "adminverify " + client.user.mention,
                             False)
+    _embedMessage.add_field(helpMessage, commandPrefix2 + "lockdown", "Sets send message permissions to false for specified role\nEx: " + commandPrefix2 + "lockdown " + "Tron", False)
```

Comment:
The parameters of the command should be in the first string, like it is for all the other commands